### ISSUE TITLE
New relevant paper(s)

### ISSUE THREAD

**Issue by tamlhp** [tamlhp/awesome-instruction-editing#13]
GenMix: Effective Data Augmentation with Generative Diffusion Model Image Editing
https://arxiv.org/abs/2412.02366

**Comment by tamlhp** [comment]
SwiftEdit: Lightning Fast Text-Guided Image Editing via One-Step Diffusion
https://arxiv.org/abs/2412.04301

**Comment by HolmesShuan** [comment]
FireFlow: Fast Inversion of Rectified Flow for Image Semantic Editing
https://arxiv.org/abs/2412.07517

**Comment by tamlhp** [comment]
PrEditor3D: Fast and Precise 3D Shape Editing
https://arxiv.org/abs/2412.06592

**Comment by tamlhp** [comment]
Diffusion-Based Attention Warping for Consistent 3D Scene Editing
https://arxiv.org/abs/2412.07984

**Comment by tamlhp** [comment]
FluxSpace: Disentangled Semantic Editing in Rectified Flow Transformers
https://arxiv.org/abs/2412.09611

**Comment by tamlhp** [comment]
MIVE: New Design and Benchmark for Multi-Instance Video Editing
https://arxiv.org/abs/2412.12877

**Comment by tamlhp** [comment]
UIP2P: Unsupervised Instruction-based Image Editing via Cycle Edit Consistency
https://arxiv.org/abs/2412.15216

**Comment by tamlhp** [comment]
CA-Edit: Causality-Aware Condition Adapter for High-Fidelity Local Facial Attribute Editing
https://arxiv.org/abs/2412.13565

**Comment by tamlhp** [comment]
Unsupervised Region-Based Image Editing of Denoising Diffusion Models
https://arxiv.org/abs/2412.12912

**Comment by tamlhp** [comment]
DriveEditor: A Unified 3D Information-Guided Framework for Controllable Object Editing in Driving Scenes
https://arxiv.org/abs/2412.19458

**Comment by tamlhp** [comment]
MAKIMA: Tuning-free Multi-Attribute Open-domain Video Editing via Mask-Guided Attention Modulation
https://arxiv.org/abs/2412.19978

**Comment by tamlhp** [comment]
Edicho: Consistent Image Editing in the Wild
https://arxiv.org/abs/2412.21079

**Comment by tamlhp** [comment]
Edit as You See: Image-guided Video Editing via Masked Motion Modeling
https://arxiv.org/abs/2501.04325

**Comment by tamlhp** [comment]
Exploring Optimal Latent Trajetory for Zero-shot Image Editing
https://arxiv.org/abs/2501.03631

**Comment by tamlhp** [comment]
FramePainter: Endowing Interactive Image Editing with Video Diffusion Priors
https://arxiv.org/abs/2501.08225

**Comment by tamlhp** [comment]
EditSplat: Multi-View Fusion and Attention-Guided Optimization for View-Consistent 3D Scene Editing with 3D Gaussian Splatting
https://arxiv.org/abs/2412.11520

**Comment by tamlhp** [comment]
CAD-Editor: A Locate-then-Infill Framework with Automated Training Data Synthesis for Text-Based CAD Editing
https://arxiv.org/abs/2502.03997

**Comment by griffith-data-analytics** [comment]
Energy-Guided Optimization for Personalized Image Editing with Pretrained Text-to-Image Diffusion Models
https://arxiv.org/abs/2503.04215

**Comment by griffith-data-analytics** [comment]
MRHaD: Mixed Reality-based Hand-Drawn Map Editing Interface for Mobile Robot Navigation
https://arxiv.org/abs/2504.00580

**Comment by griffith-data-analytics** [comment]
ScanEdit: Hierarchically-Guided Functional 3D Scan Editing
https://arxiv.org/abs/2504.15049

**Comment by griffith-data-analytics** [comment]
Early Timestep Zero-Shot Candidate Selection for Instruction-Guided Image Editing
https://arxiv.org/abs/2504.13490

**Comment by tamlhp** [comment]
Vidi: Large Multimodal Models for Video Understanding and Editing
https://arxiv.org/abs/2504.15681

**Comment by tamlhp** [comment]
Rethinking Score Distilling Sampling for 3D Editing and Generation
https://arxiv.org/abs/2505.01888

**Comment by tamlhp** [comment]
S2Edit: Text-Guided Image Editing with Precise Semantic and Spatial Control
https://arxiv.org/abs/2507.04584

**Comment by phkhanhtrinh23** [comment]
I have updated the information. May I close this issue.
Thank you everyone for your contribution!
